src/main/java/teammates/ui/pagedata/InstructorCoursesPageData.java

```diff
@@ -83,6 +83,17 @@ public RecoveryCoursesTable getRecoveryCourses() {
         return recoveryCourses;
     }
 
+    public boolean isInstructorAllowedToModify() {
+        for (int i = 0; i < getRecoveryCourses().getRows().size(); i++) {
+            RecoveryCoursesTableRow course = getRecoveryCourses().getRows().get(i);
+            if (!instructorsForCourses.get(course.getCourseId())
```

Comment:
Can we use a stream for this method? You can use `anyMatch`.

Reply:
I'll use `allMatch` instead to avoid multiple `!`s :)